variants/frontend-stimulus/hello_controller.ts

```diff
@@ -0,0 +1,18 @@
+import { Controller } from '@hotwired/stimulus';
+
+// See https://github.com/orgs/stimulus-components/repositories for examples of
+// Stimulus controllers. Consider whether it is better for the long term health
+// of your app to use these components as a dependency or take some copy & paste
+// inspiration.
+
+// To see this controller in action, add the following to the DOM:
+//
+//   <div data-controller="hello"></div>
+
+// Stimulus: https://stimulus.hotwired.dev/
+// Stimulus & Typescript: https://stimulus.hotwired.dev/reference/using_typescript/
+export default class HelloController extends Controller<Element> {
+  connect() {
+    console.log('Hello from the HelloController');
+  }
+}
```

Comment:
@G-Rath I'm linking to the Stimulus Typescript docs and have changed the example method to be more generic so I think this might be ok now?

Reply:
Yup, they merged the change for that (we were discussing it in Slack this morning, and have confirmed on a project that it is correct and a drop-in replacement)